core/frontend/src/EnvironmentDecorations.ts

```diff
@@ -86,11 +86,17 @@ export class EnvironmentDecorations {
 
   public decorate(context: DecorateContext): void {
     const env = this._environment;
+
+    let sky: RenderGraphic | undefined = undefined;
     if (env.displaySky && this._sky.params) {
-      const sky = IModelApp.renderSystem.createSkyBox(this._sky.params);
-      if (sky)
-        context.setSkyBox(sky);
+      sky = IModelApp.renderSystem.createSkyBox(this._sky.params);
     }
+    if (env.displayAtmosphere && env.atmosphere) { // Atmosphere can only be drawn if sky is also drawn
```

Comment:
`env.atmosphere` is always `true`.

Reply:
Comment is no longer accurate.

Reply:
Removed `env.atmosphere` from the check
I also worded that comment poorly, so I just removed it as extraneous info